spyderlib/utils/introspection/jedi_plugin.py

```diff
@@ -267,7 +266,7 @@ def preload(self):
     source_code = "import n"
     completions = p.get_completions(CodeInfo('completions', source_code,
         len(source_code)))
-    assert 'numpy' in completions
+    assert ('numpy', 'module') in completions
```

Comment:
Ok, I just saw that you added a new (M) for modules.

@goanpeca, what do you think about it? Should we leave it or use a (P) one?